ImageItem/ImageItem.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * Agate styled ImageItem.
+ *
+ * @module agate/ImageItem
+ * @exports ImageItem
+ * @exports ImageItemBase
+ * @exports ImageItemDecorator
+ */
+
+import kind from '@enact/core/kind';
+import {ImageItem as UiImageItem} from '@enact/ui/ImageItem';
+import PropTypes from 'prop-types';
+import compose from 'ramda/src/compose';
+import React from 'react';
+import Spottable from '@enact/spotlight/Spottable';
+import {MarqueeDecorator, MarqueeController} from '@enact/ui/Marquee';
+
+import ImageBase from '../Image';
+import Skinnable from '../Skinnable';
+
+import componentCss from './ImageItem.module.less';
+
+const MarqueeImageItem = MarqueeDecorator(UiImageItem);
+
+const ImageItemBase = kind({
```

Comment:
We should add JSDoc for the `ImageItemBase` because it is exported in `ImageItem`.